guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json

```diff
@@ -329,6 +329,10 @@
                     "name"  : "sftp-hostname",
                     "type"  : "TEXT"
                 },
+                {
+                    "name"  : "sftp-host-key",
```

Comment:
Is there a reason behind the difference in parameter order across protocols? I see `hostname`/`port`/**`host-key`** for SSH but `sftp-hostname`/**`sftp-host-key`**/`sftp-port` for VNC/RDP.

Reply:
No, just a mistake.  Reordered.